controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java

```diff
@@ -904,6 +905,25 @@ public void checkTransactionState(TxnRequest request, StreamObserver<TxnState> r
                 responseObserver, requestTag);
     }
 
+    @Override
+    public void listCompletedTransactions(Controller.ListCompletedTxnRequest request, StreamObserver<Controller.ListCompletedTxnResponse> responseObserver) {
```

Comment:
NIT: you can import the class `ListCompletedTxnRequest` and `ListCompletedTxnResponse`